process.on('uncaughtException', function (err) {
        console.debug('Caught exception: ' + err);
    });

var debug = require('debug')('PULSBS');
var app = require('../app');

app.set('port', 3100);

var server = app.listen(app.get('port'), function() {
    console.log('Pandemic University Lecture Seat Booking System server listening on port ' + server.address().port);	
	app.io.attach(server);
});

